feat(debugging): add some preconfigured .vscode debugging with docs to configure

PeterFarber opened the pull request.

PeterFarber committed:
feat(debugging): add some preconfigured .vscode debugging with docs to configure

twilson63 commented:
I am unable to get debugging to work, could you provide an example with

```erl
cu_test:full_push_test().
```

thx

PeterFarber commented:
> I am unable to get debugging to work, could you provide an example with
> 
> ```erlang
> cu_test:full_push_test().
> ```
> 
> thx

Updated the debugger config and added a youtube video tutorial.
